seml/start.py

```diff
@@ -134,8 +132,16 @@ def start_slurm_job(collection, exp_array, unobserved=False, post_mortem=False,
 
     # Construct Slurm script
     template = pkg_resources.resource_string(__name__, "slurm_template.sh").decode("utf-8")
+    if 'working_dir' in exp_array[0][0]['seml']:
+        working_dir = exp_array[0][0]['seml']['working_dir']
+    elif 'project_root_dir' in exp_array[0][0]['seml']:
+        working_dir = exp_array[0][0]['seml']['project_root_dir']
```

Comment:
Why do we have this option? Doesn't backwards compatibility only require the last branch, i.e. `SLURM_SUBMIT_DIR`?

Reply:
I was thinking about the case of the current version. There we have defined `project_root_dir`, but not `working_dir `. I guess for this case we have to also consider `exp['executable_rel']`, which we don't use anymore, for backward compatibility. I'll include that in a new commit.

Reply:
The master branch just uses `${SLURM_SUBMIT_DIR}` and this PR uses `working_dir`, so those are imho the only 2 options, are they not?